hack/e2e-test.sh

```diff
@@ -20,7 +20,6 @@ set -o pipefail
 
 SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
 ROOT_DIR="$SOURCE_DIR/.."
-export E2E_TEST_IMAGE=gcr.io/k8s-staging-perf-tests/sleep:v0.1.0
```

Comment:
Ah, we still need it here to pull image https://github.com/kubernetes-sigs/kueue/blob/f5ddc57bb4435542aad0d32a1199aca13fd88c91/hack/e2e-test.sh#L52.